### ISSUE TITLE
Disconnected people

### ISSUE THREAD

**Issue by mattuys** [Gerrygames/ViaRewind#72]
Some people have trouble joining my server. After a few seconds after logging in, they are disconnected with the message:"Internal Exception: java. io. IOException: An existing connection was forcibly closed by the remote host". The server is in 1.12.2 and the people are in 1.7.10 with Forge. I've activated the viaversion debug mode. Here are the logs:
[latest.log](https://github.com/Gerrygames/ViaRewind/files/1753235/latest.log)

**Comment by Gerrygames** [comment]
Could you try using the latest development version of ViaBackwards? https://ci.viaversion.com/view/ViaBackwards/job/ViaBackwards-DEV/58/

**Comment by mattuys** [comment]
There is no change, the problem persists.

**Comment by Gerrygames** [comment]
The user seems to not have lost connection in the log.

**Comment by mattuys** [comment]
Je précise le post de départ, les personnes étant juste en 1.7.0 sans Forge, sont déconnectés MAIS seulement quelques joueurs. Do you think this is due to server configuration and not ViaRewind??

**Comment by FlameHeroDev** [comment]
Same issue over here

**Comment by koopakondra** [comment]
I have a very similar issue and I would like to know what to do: did any of you fix this as the author did not seem to follow up as this issue is open.

**Comment by SulphurSlap** [comment]
yeah, i have a 1.12.2 server and i crash almost every time i log in on 1.7.2 or 1.8, its fine on 1.9 and above, just 1.7 and 1.8 are broken

**Comment by Gerrygames** [comment]
1. Try using the ViaBackwards build mentioned above. 
2. Send me the disconnect message of the client and the server log (set debug=true in server.properties).
3. Test if this issue is caused by some other plugin.
